datasources/src/test/scala/com/normation/plugins/datasources/UpdateHttpDatasetTest.scala

```diff
@@ -841,8 +843,8 @@ class UpdateHttpDatasetTest extends Specification with BoxSpecMatcher with Logga
       }
     }
     "have a working 'default value property' option" in {
-      test404prop(propName = "test-404", initValue = Some("test-404"), onMissing = MissingNodeBehavior.DefaultValue(JString("foo")), expectMod = true) { props =>
-        props must havePairs( props.keySet.map(x => (x, Some(JString("foo"))) ).toSeq:_* )
+      test404prop(propName = "test-404", initValue = Some("test-404"), onMissing = MissingNodeBehavior.DefaultValue(GenericProperty.parseValue("foo").toOption.get), expectMod = true) { props =>
+        props must havePairs( props.keySet.map(x => (x, Some(GenericProperty.parseValue("foo").toOption.get)) ).toSeq:_* )
```

Comment:
same for all strings here: `"foo".toConfigValue` once `GenericProperty._` is imported